clang/lib/Headers/ptrauth.h

```diff
@@ -15,6 +15,29 @@ typedef enum {
   ptrauth_key_asib = 1,
   ptrauth_key_asda = 2,
   ptrauth_key_asdb = 3,
+
+#ifdef __APPLE__
```

Comment:
I believe these are defaults for ELF as well. So, I'd remove ifdef and decide on necessary gating when there will be a precedent.

Reply:
Great, let's do that;  I also removed the comments that talk about enablement policy, that seems like a separate problem if this is to be generic